Assets/SEE/Controls/KeyBindings.cs

```diff
@@ -101,6 +116,11 @@ internal static void PrintBindings()
         /// </summary>
         internal static readonly KeyCode ToggleMenu = Register(KeyCode.Space, Scope.Always, "Turns on/off the player-action menu.");
 
+        /// <summary>
+        /// Turns on the settings menu.
+        /// </summary>
+        internal static readonly KeyCode ActivateSettings = Register(KeyCode.Escape, Scope.Always, "Turns on the settings menu.");
```

Comment:
It would be great if this didn't just turn on the settings menu, but toggles it instead (i.e., pressing the button when the menu is already open leads to it closing). Currently, the menu opens twice in such cases.